compiler/build.gradle

```diff
@@ -0,0 +1,84 @@
+plugins {
+    id 'org.jetbrains.kotlin.jvm' version'1.3.61'
+    id 'com.google.protobuf' version '0.8.11'
+    id 'org.jetbrains.dokka' version '0.10.1'
+
+    // Generate IntelliJ IDEA's .idea & .iml project files
+    // Starting with 0.8.4 of protobuf-gradle-plugin, *.proto and the gen output files are added
+    // to IntelliJ as sources. It is no longer necessary to add them manually to the idea {} block
+    // to jump to definitions from Java and Kotlin files.
+    // For best results, install the Protobuf and Kotlin plugins for IntelliJ.
+    id 'idea'
+
+    // Provide convenience executables for trying out the examples.
+    id 'application'
+    id 'maven-publish'
+}
+
+description = 'The protoc plugin for gRPC Kotlin'
+mainClassName = 'io.grpc.kotlin.generator.GeneratorRunner'
+
+repositories {
+    google()
+    jcenter()
+    mavenCentral()
+    mavenLocal()
+}
+
+// Feel free to delete the comment at the next line. It is just for safely
+// updating the version in our release process.
+def grpcVersion = '1.26.0' // CURRENT_GRPC_VERSION
+
+dependencies {
+    // Grpc and Protobuf
+    compile project(':grpc-kotlin-stub')
+    compile 'com.google.protobuf:protobuf-java:3.11.0'
+    compile "io.grpc:grpc-netty-shaded:${grpcVersion}"
+
+    // Misc
+    compile 'com.squareup:kotlinpoet:1.5.0'
+    compile "org.jetbrains.kotlin:kotlin-reflect:1.3.61"
+    compile "com.google.truth:truth:1.0.1"
+
+    // Testing
+    testImplementation "com.google.guava:guava:28.2-jre"
+    testImplementation "com.google.jimfs:jimfs:1.1"
+    testImplementation 'com.google.protobuf:protobuf-gradle-plugin:0.8.11'
+    testImplementation "com.nhaarman.mockitokotlin2:mockito-kotlin:2.2.0"
+    testImplementation "org.junit.jupiter:junit-jupiter-engine:5.5.2"
+    testImplementation "org.mockito:mockito-core:2.28.2"
+}
+
+protobuf {
+    protoc { artifact = 'com.google.protobuf:protoc:3.11.0' }
+    plugins {
+        // Specify protoc to generate using kotlin protobuf plugin
+        grpc {
+            artifact = 'io.grpc:protoc-gen-grpc-java:1.27.0'
```

Comment:
We might want a named variable for the grpc-java version too. Also, it would be good to know why this version doesn't match the `grpcVersion`.

Reply:
I think it actually should match, so I'll update that as well

Reply:
Updated!